golang/cosmos/app/app.go

```diff
@@ -359,6 +361,10 @@ func NewAgoricApp(
 		tkeys:             tkeys,
 		memKeys:           memKeys,
 	}
+	// The VM is entitled to full awareness of runtime configuration.
+	if resolvedConfig, ok := appOpts.(*viper.Viper); ok {
+		app.resolvedConfig = resolvedConfig.AllSettings()
+	}
```

Comment:
I would strongly prefer to carve out a subsection of the app config, likely named "swingset" and only pass that section through. I agree that JS and golang can share that subsection, but I really don't see a reason to overexpose the full cosmos config to JS.

Reply:
I'm convinced. Updated.